QUICKSTART.md

```diff
@@ -50,11 +50,11 @@ Congratulations! You have deployed a TiDB cluster! You can see messages in your
 
 You can now test your TiDB server using one of the following methods:
 
-- Use the MySQL client to connect to TiDB to read and write data:
+- Use the MySQL client to connect to TiDB:
 
     ```
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
-
+    You can [try TiDB](try-tidb.md) to explore the SQL statements.
```

Comment:
Please leave a blank line after line 58, to avoid possible display error on the website.